src/api/source/mvt.js

```diff
@@ -83,6 +83,9 @@ export default class MVT extends Base {
 
 
     _decodeMVTLayer(mvtLayer, metadata, mvt_extent) {
+        if (!mvtLayer.length) {
+            return { properties: [], geometries: {} };
```

Comment:
Eing? How does that prevent the problem of accessing the first feature?

Reply:
because if the layer comes empty the function returns and `_decode` is never called

Reply:
sorry, I meant:

the funcion `_autoDiscoverType` is never called

Reply:
:+1: